acc_test/resource_redis_test.go

```diff
@@ -44,7 +44,6 @@ func TestRedisResource(t *testing.T) {
 					testCheckResourceValid(resourceName),
 					testCheckResourceCreated(resourceName, hostname, username, apiKey),
 					checkClusterRunning(resourceName, hostname, username, apiKey),
-					testCheckContactIPCorrect(resourceName, hostname, username, apiKey, 5, 5),
```

Comment:
this line tests whether the cluster is allocated across 5 AZs
Alwyn said this is a test for Cassandra but not needed for Redis, after discussing with ChrisC